backend/src/index.ts

```diff
@@ -66,6 +74,8 @@ app.get("/", (req: Request, res: Response) => {
   res.send("Hello, TypeScript with Express :)))!");
 });
 
+//Will refactor this soon, I think prisma throws its own error when some

```

Comment:
backend validation separate from the database is a good idea for security reasons amongst other things. i'd recommend both, or at least certainly keeping it on the backend